shared/tinyusb/mp_usbd.h

```diff
@@ -67,6 +67,8 @@ extern const uint8_t mp_usbd_builtin_desc_cfg[MP_USBD_BUILTIN_DESC_CFG_LEN];
 
 void mp_usbd_task_callback(mp_sched_node_t *node);
 
+bool tusb_inited(void);
```

Comment:
This shouldn't be needed, it's already defined in `tusb.h`.

Reply:
OK. For the same reason, the declaration of `tusb_init()` further down is also not needed.